PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamEvent.h

```diff
@@ -192,11 +192,25 @@ class AliFemtoDreamEvent {
     return fspher;
   }
   ;
+  void SetSphero(double sphero) {
+    fsphero = sphero;
+  }
+  ;
+  float GetSphero() const {
+    return fsphero;
+  }
+  ;
```

Comment:
these ; are actually not needed here :)
They are already contained within the function so that's good enough